aws-logs-subscriptionfilter/example_inputs/inputs_1_delete.json

```diff
@@ -0,0 +1,7 @@
+{
+  "desiredResourceState": {
+    "FilterName": "878a189a-8e29-4a69-be48-2e5-rqxT4po09X3Y",
```

Comment:
Only `inputs_1_create.json`, `inputs_1_invalid.json` and `inputs_1_update.json` are required for the contract tests. 
`inputs_1_delete.json` and `inputs_1_list.json` are not used.